tequilapi/endpoints/nat.go

```diff
@@ -50,9 +58,32 @@ func (ne *NATEndpoint) NATStatus(resp http.ResponseWriter, _ *http.Request, _ ht
 	utils.WriteAsJSON(ne.stateProvider.GetState().NATStatus, resp)
 }
 
+// NATType provides NAT type in terms of traversal capabilities
+// swagger:operation GET /nat/type NAT NATTypeDTO
+// ---
+// summary: Shows NAT type in terms of traversal capabilies
+// description: Returns NAT type. May produce invalid result while VPN connection is established
+// responses:
+//   200:
```

Comment:
Only 200 status is described, but based on code 500 is possible too.

Reply:
Updated